classes/class-gf-securesubmit.php

```diff
@@ -2666,13 +2656,17 @@ private function getHpsServicesConfig($key)
     {
         static $config = null;
         if (empty($config)) {
-            $config = new HpsServicesConfig();
+            /*$config = new HpsServicesConfig();
             $config->secretApiKey = $key;
             $config->developerId = '002914';
-            $config->versionNumber = '1916';
+            $config->versionNumber = '1916';*/
+            $config = new ServicesConfig();
+            $config->secretApiKey = $key;
+            $config->serviceUrl = "https://cert.api2.heartlandportico.com";
```

Comment:
The configuration needs to support production as well. We can check the key to determine which environment should be used.

Reply:
updated as per review comment